src/Symfony/Component/HttpKernel/DataCollector/ConfigDataCollector.php

```diff
@@ -288,4 +285,44 @@ private function determineSymfonyState()
 
         return $versionState;
     }
+
+    /**
+     * Gets the names, paths and installed versions of the bundles enabled in
+     * the application.
+     *
+     * @return array
+     */
+    private function getBundleData()
+    {
+        $bundles = array();
+
+        try {
+            $composerLockPath = $this->kernel->getRootDir().'/../composer.lock';
+            $composerLock = json_decode(file_get_contents($composerLockPath), true);
+            foreach (array_merge($composerLock['packages'], $composerLock['packages-dev']) as $dependency) {
+                $dependencyNameParts = explode('/', $dependency['name']);
+                $installedDependencies[$dependencyNameParts[1]] = $dependency['version'];
+            }
+        } catch (\Exception $e) {
+            $installedDependencies = array();
+        }
+
+        foreach ($this->kernel->getBundles() as $name => $bundle) {
+            // Transform bundle name into dependency name: 'DoctrineFixturesBundle' -> 'doctrine-fixtures-bundle'
+            $dependencyName = strtolower(preg_replace('/(?<=\\w)([A-Z])/', '-$1', $name));
```

Comment:
Maybe you should get the dependency name based of the bundle path instead of trying to translate the bundle name into a package name. This would solve all the special cases (Sensio bundles, bundles installed with symfony/symfony, etc ...).